src/Footer/style.scss

```diff
@@ -0,0 +1,74 @@
+@import './_variables.scss';
+
+.info {
+  display: flex;
+  padding: 50px 60px 70px;
+  text-align: center;
+  background-attachment: fixed;
+  background-image: url('../image/background-foo.png');
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: cover;
+
+
+  h1 {
+    margin-bottom: 20px;
+    font-size: 21px;
+    font-weight: bold;
+    color: $white;
+  }
+
+  p {
+    margin-top: 20px;
+    font-size: 16px;
+    line-height: 1.5;
+    color: $white;
+
+  }
+
+  span {
+    font-weight: bold;
+  }
+}
+
+.About {
+  margin-left: 15%;
+}
+
+.open {
+  margin-left: 160px;
+}
+
+.Our-Location {
+  margin-left: 190px;
```

Comment:
too big margin! are u sure in both of them??

Reply:
not really ,so i  have 
changed it to 100px

Reply:
still to much
I will double check it on my local